docs/process/release.md

```diff
@@ -53,17 +53,32 @@ In general, cherry picks for LTS should only be done if it meets one of the crit
 
 # Release
 
+Releasing is performed using Angular's unified release tooling. Each week, two releases are expected, `latest` and `next` on npm.
+
+The `latest` release should be done first (typically a patch release), followed by the `next` release.
```

Comment:
Is this a hard requirement by the tooling? If yes, then we should probably use stronger language here. If not, then we probably shouldn't have this line at all.